tuning-crc-openshift-cluster/enable-alpha-api.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+${SSH_CMD} sudo cat /etc/kubernetes/manifests/kube-apiserver-pod.yaml  > current_kubeapiserver_manifest.json
+current_args=`cat current_kubeapiserver_manifest.json | jq -r '.spec.containers[]  | select(.name == "kube-apiserver") | .args[0]'`
+additional_args=' --runtime-config=settings.k8s.io/v1alpha1=true --enable-admission-plugins=NamespaceAutoProvision,MutatingAdmissionWebhook,PodPreset '
+new_args="${current_args} ${additional_args}"
+echo $new_args
+${JQ}  --arg new_args "$new_args" '(.spec.containers[] | select(.name == "kube-apiserver") | .args[0]) |= $new_args' current_kubeapiserver_manifest.json > updated_kubeapiserver_manifest.json
+cat updated_kubeapiserver_manifest.json | ${JQ} -c '.' > unformatted_updated_kubeapiserver_manifest.json
```

Comment:
Please recheck if kube-api manifest being updated 'does' enable 'alpha/api' and Podpresets.

I suspect '.' is not required to create an unformatted manifest version.

Reply:
> I suspect '.' is not required to create an unformatted manifest version.
For me it's required:
```
$ ssh ... cat /etc/kubernetes/manifests/kube-apiserver-pod.yaml | jq -c >/tmp/foo.txt
jq - commandline JSON processor [version 1.5]
[...]

$ ssh ... cat /etc/kubernetes/manifests/kube-apiserver-pod.yaml | jq -c . >/tmp/foo.txt
$
```

Without the `.`, `jq -c` errors out when it's redirected to a file, https://stackoverflow.com/questions/48964305/write-output-to-a-file-after-piped-to-jq has some details about this.

Reply:
> Please recheck if kube-api manifest being updated 'does' enable 'alpha/api' and Podpresets.

`/etc/kubernetes/manifests/kube-apiserver-pod.yaml` does contain ` --runtime-config=settings.k8s.io/v1alpha1=true --enable-admission-plugins=NamespaceAutoProvision,MutatingAdmissionWebhook,PodPreset` ( https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/code-ready_snc/246/#1:pull-ci-code-ready-snc-master-e2e-snc/1316851898141118464/build-log.txt%3A2091 ) but:
```
$ ./oc api-resources --api-group=settings.k8s.io
NAME   SHORTNAMES   APIGROUP   NAMESPACED   KIND 
```